cmake/SeracMacros.cmake

```diff
@@ -24,44 +24,28 @@
 macro(serac_add_code_checks)
 
     set(options)
-    set(singleValueArgs PREFIX )
-    set(multiValueArgs  INCLUDES EXCLUDES)
+    set(singleValueArgs PREFIX)
+    set(multiValueArgs)
 
     # Parse the arguments to the macro
     cmake_parse_arguments(arg
          "${options}" "${singleValueArgs}" "${multiValueArgs}" ${ARGN})
 
-    set(_all_sources)
+    # Create file globbing expressions that only include directories that contain source
+    set(_base_dirs "tests" "src" "examples")
```

Comment:
does this add `examples` to be code-checked? was it not previously? (just testing my understanding)

Reply:
I believe yes, hence the small style updates.